network-api/networkapi/buyersguide/templates/about/methodology.html

```diff
@@ -12,19 +12,21 @@ <h1 class="h2-heading mb-4">
   <section class="methodology-intro">
     <p class="body">
       {% blocktrans trimmed %}
-      The goal of Mozilla’s <em>*Privacy Not Included</em> buyer’s guide is to help consumers shop smart — and safe — for
-      products that connect to the Internet. Many companies don’t provide clear information about the privacy and
-      security of the connected devices they sell. Is your personal data being used in ways you may not have
-      anticipated or expected? What are the risks of buying a product that collects your health data? How does the
-      company regularly test for and fix security vulnerabilities?
+      The goal of Mozilla’s <em>*Privacy Not Included</em> buyer’s guide is to help consumers
+      shop smart — and safe — for products that connect to the Internet. Many companies don’t
```

Comment:
we'll need to align this with our overall "how do we use em dashes" policy

Reply:
@TheoChevalier thoughts?